doc/gitlab.nvim.txt

```diff
@@ -565,36 +607,94 @@ reviewer when checking out a new branch:
+    vim.keymap.set("n", "k", [[<Cmd>call search('[] @', 'b')<CR>]], { buffer = 0, desc = "Go to previous node" })
 <
 
+Reviewer keybindings ~
+
+Most of the keybindings `gitlab.nvim` sets are normal mode mappings, with the
+exception of `keymaps.reviewer.create_comment` and
+`keymaps.reviewer.create_suggestion` which work in both normal and visual
+mode. In normal mode, these keybindings are |operator|s that accept a |motion|
+(with an optional |count|). E.g., `c2j` will create a comment for the current
+and the next 2 lines. Similarly, `sip` will create a suggestion for the "inner
+paragraph". The operator forces |linewise| visual selection, so it works
+correctly even if the motion itself works |characterwise| (e.g., |i(| for
+selecting the inner parentheses block).
+
+The keybindings also work in visual mode, e.g., if you first want to make
+sure you are commenting on the right text segment/object, you can do `v4j` to
+visually select the current and the next 4 lines, followed by either `c` for a
+normal comment or `s` for a suggestion.
+
+To create a comment or suggestion for the current line, just duplicate the
+keybinding: `cc` and `ss`. Alternatively, you can use any motion that moves on
+that line only, e.g., `cw`, `cl`, `c$`, etc.
+
+The plugin takes advantage of the fact that the `c` and `s` normal mode
+commands modify the text, but the Diffview buffers are set as |nomodifiable|,
+so the keys are practically free for remapping. If, however, you don't like
+the default `c` and `s` keybindings, you can remap these operators to a
+different key or keys, e.g. `keymaps.reviewer.create_comment = "<leader>c"`.
+This will still work for commenting on the same line by duplicating the
+keybinding, e.g., `<leader>c<leader>c`.
+
+Delay in keybindings ~
+
+If you are experiencing a lag in some of the keybindings, it may be the case
+that they have the same prefix with another existing keybinding. E.g., you
+normally remap `co` to call function F, then the default `c` operator mapping
+in the reviewer will take some time to trigger. If 'showcmd' is set, then
+after pressing `c`, you'll first see a `c` in the right bottom corner of the
+screen, and during that time `neovim` will also accept `o` as the next
+character and it will allow you to call function F. However, if the
+'timeoutlen' passes without you pressing any keys, the `c` will change to |g@|
+and `neovim` will be waiting for you to press a {motion} keybinding. You will
+no longer be able to finish your `co` mapping. If the lag annoys you and you
+don't really care about your `co` mapping for this buffer, you can force the
+`gitlab.nvim` mapping to fire immediately by adding a special field into your
+`keymaps` table. E.g., if the name of the keybinding is
+`reviewer.create_comment`, then you can add the following:
+>lua
+    keymaps = { reviewer = { create_comment_nowait = true }}
+<
+This works by adding the |nowait| argument to the |vim.keymap.set| call.
+
+
```

Comment:
I actually find the possibility to use `nowait` useful, so I'd be grateful if it could stay there. E.g., I use [debugprint](https://github.com/andrewferrier/debugprint.nvim/) and don't want to change it's default `g?` prefix for keybindings, but at the same time, I don't want to change `g?` for help in `gitlab.nvim` and I don't want to wait for it to apply.

Other plugins also make it possible to adjust `nowait` on their keymaps, e.g., [neo-tree](https://github.com/nvim-neo-tree/neo-tree.nvim), or [diffview](https://github.com/sindrets/diffview.nvim). So this I don't think adding this possibility to `gitlab.nvim` is somehow strange.

But I agree that the description was too much, so I've simplified it.

Reply:
Thank you, the simpler description in the help document makes me feel better about this change. I was not familiar with other plugins' use of this feature.